content-warehouse/src/main/java/contentwarehouse/v1/UpdateDocumentSchema.java

```diff
@@ -0,0 +1,104 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contentwarehouse.v1;
+
+// [START contentwarehouse_update_document_schema]
+
+import com.google.cloud.contentwarehouse.v1.DocumentSchema;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaName;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceSettings;
+import com.google.cloud.contentwarehouse.v1.PropertyDefinition;
+import com.google.cloud.contentwarehouse.v1.TextTypeOptions;
+import com.google.cloud.contentwarehouse.v1.UpdateDocumentSchemaRequest;
+import com.google.cloud.resourcemanager.v3.Project;
+import com.google.cloud.resourcemanager.v3.ProjectName;
+import com.google.cloud.resourcemanager.v3.ProjectsClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class UpdateDocumentSchema {
+  public static void updateDocumentSchema() throws IOException, 
+        InterruptedException, ExecutionException, TimeoutException { 
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region"; // Format is "us" or "eu".
+    String documentSchemaId = "your-document-schema-id";
+    updateDocumentSchema(projectId, location, documentSchemaId);
+  }
+
+  public static void updateDocumentSchema(String projectId, String location, 
+        String documentSchemaId) throws IOException, InterruptedException,
+          ExecutionException, TimeoutException { 
+    String projectNumber = getProjectNumber(projectId);
+
+    String endpoint = "contentwarehouse.googleapis.com:443";
+    if (!"us".equals(location)) {
+      endpoint = String.format("%s-%s", location, endpoint);
+    }
+
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+             DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+
+    // Create the Schema Service Client 
+    try (DocumentSchemaServiceClient documentSchemaServiceClient = 
+            DocumentSchemaServiceClient.create(documentSchemaServiceSettings)) {
+            
+      //The full resource name of the location, e.g.: 
+      //projects/{project_number}/location/{location}/documentSchemas/{document_schema_id}
+      DocumentSchemaName documentSchemaName = 
+          DocumentSchemaName.of(projectNumber, location, documentSchemaId);
+            
+      // Define the new Schema Property with updated values
+      PropertyDefinition propertyDefinition = PropertyDefinition.newBuilder()
+          .setName("stock_symbol")
+          .setDisplayName("Searchable text")
+          .setIsSearchable(true)
+          .setIsRepeatable(true)
+          .setIsRequired(false)
+          .setTextTypeOptions(TextTypeOptions.newBuilder().build()).build();
```

Comment:
Does the empty builder need to be set here?

Reply:
Setting text type options is required for this document schema. Should I move this out of the property definition builder into it's own object?

Reply:
Will it be easier for the customer to understand why the builder is empty, if this is not viewed in the context of the quickstart?

Each of our sample should be standalone and complete on it's own. So, if it exists to match Quickstart, then could we refactor this sample to make it independent?

Reply:
Yeah I can refactor this one. Do you think it would be good to show creating a document schema, pulling the document schema ID, then updating the document schema? I think this would help with the customer's understanding

Reply:
Yes, that sounds like a good plan.
Kindly make sure each snippet is in it's own file. Thanks

Reply:
Just pushed these changes

Reply:
@chasinandrew  Thanks for the changes. Can we now go ahead to remove the empty builder?